debug_gym/gym/tools/agent.py

```diff
@@ -0,0 +1,81 @@
+import logging
+
+from debug_gym.agents.history_tracker import build_history_prompt
+from debug_gym.gym.entities import Observation
+from debug_gym.gym.tools.tool import EnvironmentTool
+from debug_gym.gym.tools.toolbox import Toolbox
+from debug_gym.gym.utils import is_subdirectory, show_line_number
+from debug_gym.llms.base import LLM
+
+
+@Toolbox.register()
+class AgentTool(EnvironmentTool):
+    name: str = "agent"
+    examples = [
+        """agent(query="") Executes query to send to this specialized agent.\n"""
```

Comment:
What do you mean by "Executes query".
```suggestion
        """agent(query="") Send a query to this specialized agent.\n"""
```